src/Pages/Adopt.jsx

```diff
@@ -679,6 +750,411 @@ export const Adopt = () => {
+                      {...register("what_other_pets", {
+                        required: "Other pets information is required",
+                      })}
+                      error={!!errors.what_other_pets?.message}
+                      helperText={errors.what_other_pets?.message}
+                    />
+                  </Fade>
+                )}
+              </Stack>
+              <Typography
+                variant="h3"
+                sx={{ mb: 2, mt: 4 }}
+              >
+                Bird Care
+              </Typography>
+              <Stack spacing={2}>
+                <TextField
+                  id="what-supp-info"
+                  label="What sources of information do you use to supplement your knowledge of avian care?"
+                  sx={{
+                    "& .MuiFormLabel-root": { whiteSpace: "normal" },
+                    "& .MuiFormLabel-root.MuiInputLabel-shrink": { whiteSpace: "nowrap" },
+                  }}
+                  variant="outlined"
+                  multiline
+                  minRows={4}
+                  {...register("what_supp_info", {
+                    required: "Avian sources of information is required",
+                  })}
+                  error={!!errors.what_supp_info?.message}
+                  helperText={errors.what_supp_info?.message}
+                />
+                <TextField
+                  id="lifestyle-changes"
+                  label="Please describe the lifestyle changes you might anticipate over the next 5 years? 10 years? 25 years?"
+                  sx={{
+                    "& .MuiFormLabel-root": { whiteSpace: "normal" },
+                    "& .MuiFormLabel-root.MuiInputLabel-shrink": { whiteSpace: "nowrap" },
+                  }}
+                  variant="outlined"
+                  multiline
+                  minRows={4}
+                  {...register("lifestyle_changes", {
+                    required: "Lifestyle changes description is required",
+                  })}
+                  error={!!errors.lifestyle_changes?.message}
+                  helperText={errors.lifestyle_changes?.message}
+                />
+                <TextField
+                  id="vacation-care"
+                  label="When you travel or go on an extended vacation, who will care for your bird?"
+                  sx={{
+                    "& .MuiFormLabel-root": { whiteSpace: "normal" },
+                    "& .MuiFormLabel-root.MuiInputLabel-shrink": { whiteSpace: "nowrap" },
+                  }}
+                  variant="outlined"
+                  multiline
+                  minRows={4}
+                  {...register("vacation_care", {
+                    required: "Vacation care plan is required",
+                  })}
+                  error={!!errors.vacation_care?.message}
+                  helperText={errors.vacation_care?.message}
+                />
+                <TextField
+                  id="death-plans"
```

Comment:
lol `death-plans` 😆

Reply:
lol...I'm just using Jon's field name that he set up for the backend!

Reply:
🤣